#! /bin/sh /usr/share/dpatch/dpatch-run
## 12-fix-for-fox-include.dpatch by Thomas Girard <thomas.g.girard@free.fre>,
## updated for ACE 5.7.4 by Pau Garcia i Quiles <pgquiles@elpauer.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Drop fox/ prefix in include, it's correctly set by fox-config

@DPATCH@

diff -rupd ACE_wrappers.orig/ace/FoxReactor/FoxReactor.h ACE_wrappers/ace/FoxReactor/FoxReactor.h
--- ACE_wrappers.orig/ace/FoxReactor/FoxReactor.h	2009-11-23 01:19:58.000000000 +0100
+++ ACE_wrappers/ace/FoxReactor/FoxReactor.h	2009-11-23 23:50:17.000000000 +0100
@@ -28,7 +28,7 @@
 #include "ace/Select_Reactor.h"
 #include "ace/FoxReactor/ACE_FoxReactor_export.h"
 
-#include <fox/fx.h>
+#include <fx.h>
 
 /**
  * @class ACE_FoxReactor
